src/com/sbezboro/standardplugin/listeners/PlayerChatListener.java

```diff
@@ -28,24 +41,68 @@ public void onPlayerChat(AsyncPlayerChatEvent event) {
 
 		String message = event.getMessage();
 
-		if (player.getTimeSpent() < 600 && matchesHelpPattern(message)) {
-			plugin.getLogger().info("Sending help message to " + player.getDisplayName(false));
+		if (player.getTimeSpent() < 600) {
+			if (matchesGroupHelpPattern(message)) {
+				plugin.getLogger().info("Sending group help message to " + player.getDisplayName(false));
+
+				player.sendDelayedMessages(new String[] {
+						ChatColor.GRAY + "Hey " + ChatColor.BOLD + player.getDisplayName(false) +
+								ChatColor.GRAY + ", need help with our custom plugin " +
+								ChatColor.ITALIC + "Groups" + ChatColor.GRAY + "?",
+						ChatColor.GRAY + "Try clicking on this link: standardsurvival.com/guide"
+				});
+			}
+			
+			else if (matchesProtectionHelpPattern(message)) {
+				if (player.isPvpProtected()) {
+					plugin.getLogger().info("Sending PVP protection help message to " + player.getDisplayName(false));
+					
+					player.sendDelayedMessages(new String[] {
+							ChatColor.GRAY + ChatColor.BOLD + player.getDisplayName(false) +
+									ChatColor.GRAY + ", you start off with one hour of" +
+									" PVP and hunger protection.",
+							ChatColor.GRAY + "Don't worry, you will receive several warnings" +
+									" before it expires."
+					});
+				}
+			}
+			
+			else if (matchesGenericHelpPattern(message)) {
+				plugin.getLogger().info("Sending generic help message to " + player.getDisplayName(false));
 
-			player.sendDelayedMessages(new String[] {
-					ChatColor.GRAY + "Hey " + ChatColor.BOLD + player.getDisplayName(false) +
-							ChatColor.GRAY + ", need help? Try clicking on this link:",
-					ChatColor.GRAY + "standardsurvival.com/help"
-			});
+				player.sendDelayedMessages(new String[] {
+						ChatColor.GRAY + "Hey " + ChatColor.BOLD + player.getDisplayName(false) +
+								ChatColor.GRAY + ", need help? Try clicking on this link:",
+						ChatColor.GRAY + "standardsurvival.com/guide"
```

Comment:
I used `/help` rather than `/guide` to track how effective this message was vs. people just clicking on the guide page on the website